front/app/containers/Admin/projects/project/general/components/ProjectTypePicker.tsx

```diff
@@ -22,9 +22,6 @@ export default ({ projectType, handleProjectTypeOnChange }: Props) => (
         content={<FormattedMessage {...messages.projectTypeTooltip} />}
       />
     </SubSectionTitle>
-    <StyledWarning>
-      <FormattedMessage {...messages.projectTypeWarning} />
-    </StyledWarning>
     <Radio
       className="e2e-project-type-timeline"
       onChange={handleProjectTypeOnChange}
```

Comment:
Is it worth removing the handlers from this radio button too? Just to be clear that it shouldn't do anything?